### ISSUE TITLE
ADS-B: the map is not displayed

### ISSUE THREAD

**Issue by choupette-change** [f4exb/sdrangel#677]
hello everyone, i just installed the latest version: 4.21.0. I am using the ADSB channel but my card remains desperately empty and white, where is the problem?
![image](https://user-images.githubusercontent.com/68333247/97610419-20b06a80-1a15-11eb-9759-f117081358c1.png)

**Comment by srcejon** [comment]
If this is on windows, it may be because the opensll libraries were not included in the first release. They should in the next build. You should also be able to copy them in yourself.

**Comment by srcejon** [comment]
The libraries are here: https://github.com/f4exb/sdrangel-windows-libraries/tree/master/openssl

**Comment by choupette-change** [comment]
Thanks for the answer but I'm on Ubuntu 20.04.01 LTS ...

**Comment by srcejon** [comment]
Can you attach the debug log, please?

**Comment by choupette-change** [comment]
ok, i'm new, where can i find it? I only upgraded the part https://github.com/f4exb/sdrangel/wiki/Compile-from-source-in-Linux#build-sdrangel. Do I have to reinstall all ? https://github.com/f4exb/sdrangel/wiki/Compile-from-source-in-Linux

**Comment by srcejon** [comment]
Under Preferences menu > Logging. Set log file to Debug and Check log to file. 

You might want to check you have the packages installed listed in the second link, particularly these, which are needed for the map:

libqt5quick5 \
qml-module-qtlocation  qml-module-qtlocation qml-module-qtpositioning qml-module-qtquick-window2 \
qml-module-qtquick-dialogs qml-module-qtquick-controls qml-module-qtquick-layouts \
libqt5serialport5-dev qtdeclarative5-dev qtpositioning5-dev qtlocation5-dev

**Comment by choupette-change** [comment]
it's ok, it was missing:
qml-module-qtlocation
qml-module-qtpositioning
qml-module-qtquick-window2
qml-module-qtquick-dialogs
qml-module-qtquick-controls
qml-module-qtquick-layouts
qtlocation5-dev
I installed:
--- sudo apt-get update && sudo apt-get -y upgrade
--- sudo apt-get install libqt5quick5 qml-module-qtlocation qml-module-qtpositioning qml-module-qtquick-window2 qml-module-qtquick-dialogs qml-module-qtquick-controls qml-module-qtquick-layouts libqt5serialport5-dev qtdeclarative5-dev qtpositioning5-dev qtlocation5-dev
I reinstalled: https://github.com/f4exb/sdrangel/wiki/Compile-from-source-in-Linux#build-sdrangel
Besides, can you tell me the optimum settings because I find it less sensitive than ADSBOX for example. Thank you
![image](https://user-images.githubusercontent.com/68333247/97635418-9fb69a80-1a37-11eb-8981-2cba36b7150a.png)

**Comment by f4exb** [comment]
In the next release the correlation threshold setting is going to change a little bit (see #675) so what you may learn now may change a little bit later.

**Comment by choupette-change** [comment]
Is it possible to output the information in SBS-1 format?

**Comment by srcejon** [comment]
Not currently, just Beast format. What do you need SBS-1 for?

**Comment by choupette-change** [comment]
I use it with the script on Xastir: Xastir / scripts / ads-b.pl

**Comment by choupette-change** [comment]
How to change the units into SI units: Altitude ft in m, speed kn in km / h, climb etc ...? In which files to make the modifications? thank you for the info

**Comment by f4exb** [comment]
> In which files to make the modifications?

There is no such think I guess. This has to be an option in the GUI. Although in aviation knots and feet are used it could be interesting to translate values to the metric system that may be more meaningful for people (like me) who have been dealing with meters since the childhood.

**Comment by srcejon** [comment]
It doesn't currently, but will do in the next release, hopefully ready later this week.

**Comment by srcejon** [comment]
The map isn't currently displayed on the Windows 4.21.1 release. 

There is an error in the log file: "QtQuick.Window" version 2.4 is not installed

This is because it's built using Qt 5.9.9 and according to https://doc.qt.io/qt-5/qtquickcontrols-index.html#versions Qt 5.9.x has version 2.2 of the QtQuick controls. Version 2.4 is only available with Qt >5.11. (I was using 5.13.2 for my development)

Is there any plan to upgrade Qt on Windows, or should I get it working with 5.9?

**Comment by f4exb** [comment]
Windows binaries are produced using Appveyor images. See: https://www.appveyor.com/docs/windows-images-software/ It looks like Qt 5.13.2 should be available for VS 2017 image (that is used) but may require some definition in the `.appveyor.yml` file that I am not aware of.

**Comment by f4exb** [comment]
Some contribution effort is being made to upgrade to MSVC 2019 and this image uses Qt 5.15.1. It looks like it builds. I am waiting for a PR.

**Comment by f4exb** [comment]
This https://github.com/f4exb/sdrangel/blob/master/cmake/Modules/QtLocator.cmake#L10 has to be changed so that it works with 5.1x ranges:
```
FILE(GLOB QT_VERSIONS "${QT_BIN}/5.1*")
```
Then:
```
-- Qt found: C:/Qt/5.13.2/msvc2017_64
-- Compiling for 24 bit Rx DSP chain
```

**Comment by choupette-change** [comment]
Following the installation of v4.21.1, the value 2MS/s 
![adsb_1](https://user-images.githubusercontent.com/68333247/98098869-2f70a480-1e8f-11eb-8940-e2b4db7ff06b.PNG)
does not appear ??
ADS-B: added 2MS / s sample rate

**Comment by f4exb** [comment]
https://github.com/f4exb/sdrangel/commit/ecbf85e1a08e41477d310f06246d1bb9785359db

**Comment by f4exb** [comment]
All mentioned extra pieces of functionality were implemented in this PR: #688
